web-app/src/main/webapp/index.html

```diff
@@ -78,10 +78,17 @@
       </form>
     </div>
     <!-- Bottom dock -->
-    <div class="container u-full-width" id="dock">
+    <div id="dock">
+      <div id="dock-buttons">
+        <img id="menu-button" class="small-button" src="images/options-menu.svg"
```

Comment:
This is ok for now since the buttons don't do anything, but once they do, best to wrap them inside an `<a>` tag with a `aria-role="button"`. You can also use `background-image` instead of an `<img>` tag.

See resources like https://moderncss.dev/icon-button-css-styling-guide/

Reply:
Will fix this in the next PR.